grails-app/controllers/org/pih/warehouse/api/ReplenishmentApiController.groovy

```diff
@@ -57,7 +57,7 @@ class ReplenishmentApiController {
 
         bindReplenishmentData(replenishment, currentUser, currentLocation, jsonObject)
         Order order = replenishmentService.createOrUpdateOrderFromReplenishment(replenishment)
-        if (order.hasErrors() || !order.save(flush: true)) {
+        if (order.hasErrors()) {
```

Comment:
We should not be doing that check here in the controller. I think I saw that this was moved to the service in another PR and I just wanted to mention that I think is the proper approach.